Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM ubuntu:20.04
+
+USER root
```

Comment:
Why are we using the root user as default?
Is there a reason behind this?

Reply:
not that I know of, copied from [original Dockerfile](https://github.com/valory-xyz/open-aea/blob/main/scripts/acn/Dockerfile)